README.rst

```diff
@@ -22,13 +19,6 @@ Jira Python Library
 .. image:: https://readthedocs.org/projects/jira/badge/?version=master
     :target: https://jira.readthedocs.io/
 
-.. image:: https://travis-ci.com/pycontribs/jira.svg?branch=master
-    :target: https://travis-ci.com/pycontribs/jira
```

Comment:
Might be able to at the GHA badge here:

https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge

Reply:
I considered it but decided to delay until we are sure it is stable ;)